sympy/functions/special/tests/test_error_functions.py

```diff
@@ -624,14 +624,15 @@ def test_si():
     assert Si(x).series(x, oo) == pi/2 - (- 6/x**3 + 1/x \
         + O(x**(-7), (x, oo)))*sin(x)/x - (24/x**4 - 2/x**2 + 1 \
         + O(x**(-7), (x, oo)))*cos(x)/x
-
+    
     t = Symbol('t', Dummy=True)
-    assert Si(x).rewrite(sinc).dummy_eq(Integral(sinc(t), (t, 0, x)))
-
```

Comment:
Okay wait, I'm not sure why have you removed this test. If this is already present in master, no need to disturb it